backend/utils/logger.js

```diff
@@ -1,16 +1,29 @@
 const Logger = require('logplease')
 
-Logger.setLogLevel(process.env.LOG_LEVEL || 'INFO')
+const ROOT_NAME = 'dshop'
 
-const apiLogger = Logger.create('api', {
-  showTimestamp: false
-})
+if (typeof process.env.LOG === 'undefined') {
+  Logger.setLogLevel(process.env.LOG_LEVEL || 'INFO')
+}
+
+const isProd = process.env.NODE_ENV === 'production'
+
+function getLogger(name) {
+  name = name ? `${ROOT_NAME}.${name}` : ROOT_NAME
+
+  return Logger.create(name, {
+    showTimestamp: !isProd,
+    useColors: !isProd
+  })
+}
 
-const listenerLogger = Logger.create('listener', {
-  showTimestamp: false
-})
+const rootLogger = getLogger()
+const apiLogger = getLogger('dshop.api')
```

Comment:
given line #12, isn't that going to create logger dshop.dshop.api?

Reply:
yeah, and actually, I think these can just be removed.  There's nothing else to keep compatibility with.  Thanks